ui/apps/platform/src/Containers/Collections/CollectionsTable.tsx

```diff
@@ -1,24 +1,54 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import { useHistory } from 'react-router-dom';
-import { Button, ButtonVariant, Truncate, Bullseye, Text } from '@patternfly/react-core';
+import {
+    Toolbar,
+    ToolbarContent,
+    ToolbarItem,
+    Pagination,
+    Button,
+    ButtonVariant,
+    Truncate,
+    SearchInput,
+    Bullseye,
+    Text,
+} from '@patternfly/react-core';
```

Comment:
Because prettier indentation affects changed files anyway, would you be willing to alphabetize the names?